testcontainers/tests/cli_client.rs

```diff
@@ -81,6 +84,28 @@ fn generic_image_exposed_ports() {
         .is_success());
 }
 
+#[test]
+fn generic_image_running_with_extra_hosts_added() {
+    let docker = clients::Cli::default();
+    let msg = WaitFor::message_on_stdout("server is ready");
+
+    let server_1 = GenericImage::new("simple_web_server", "latest").with_wait_for(msg.clone());
```

Comment:
This is duplicated several times in these tests now, can we fix that by extracting a function that gives us a `GenericImage` for the `simple_web_server`?